debian/rules

```diff
@@ -1,4 +1,6 @@
 #!/usr/bin/make -f
 
+export DH_VIRTUALENV_INSTALL_ROOT=/usr/share
+
 %:
-	dh $@ --with python-virtualenv --setuptools --preinstall requirements/base.txt
```

Comment:
otherwise you'd end up with following error:

```
Exception:
Traceback (most recent call last):
  File "/home/dejv/Workspace/beaver/python-beaver/debian/python-beaver/usr/share/python-beaver/local/lib/python2.7/site-packages/pip-1.1-py2.7.egg/pip/basecommand.py", line 104, in main
    status = self.run(options, args)
  File "/home/dejv/Workspace/beaver/python-beaver/debian/python-beaver/usr/share/python-beaver/local/lib/python2.7/site-packages/pip-1.1-py2.7.egg/pip/commands/install.py", line 214, in run
    InstallRequirement.from_line(name, None))
  File "/home/dejv/Workspace/beaver/python-beaver/debian/python-beaver/usr/share/python-beaver/local/lib/python2.7/site-packages/pip-1.1-py2.7.egg/pip/req.py", line 107, in from_line
    return cls(req, comes_from, url=url)
  File "/home/dejv/Workspace/beaver/python-beaver/debian/python-beaver/usr/share/python-beaver/local/lib/python2.7/site-packages/pip-1.1-py2.7.egg/pip/req.py", line 40, in __init__
    req = pkg_resources.Requirement.parse(req)
  File "/home/dejv/Workspace/beaver/python-beaver/debian/python-beaver/usr/share/python-beaver/local/lib/python2.7/site-packages/setuptools-0.6c11-py2.7.egg/pkg_resources.py", line 2510, in parse
    reqs = list(parse_requirements(s))
  File "/home/dejv/Workspace/beaver/python-beaver/debian/python-beaver/usr/share/python-beaver/local/lib/python2.7/site-packages/setuptools-0.6c11-py2.7.egg/pkg_resources.py", line 2436, in parse_requirements
    line, p, specs = scan_list(VERSION,LINE_END,line,p,(1,2),"version spec")
  File "/home/dejv/Workspace/beaver/python-beaver/debian/python-beaver/usr/share/python-beaver/local/lib/python2.7/site-packages/setuptools-0.6c11-py2.7.egg/pkg_resources.py", line 2404, in scan_list
    raise ValueError("Expected "+item_name+" in",line,"at",line[p:])
ValueError: ('Expected version spec in', 'requirements/base.txt', 'at', '/base.txt')
```

this is because pip will be executed as `pip install requirements/base.txt` instead of `pip install -r requirements/base.txt`

I reckon there is no point of having preinstall here, because setuptools handle everything properly.